stubs/protobuf/google/protobuf/internal/extension_dict.pyi

```diff
@@ -1,10 +1,14 @@
 from typing import Any, Generic, Iterator, Text, TypeVar, Union
 
 from google.protobuf.descriptor import FieldDescriptor
+from google.protobuf.internal.containers import RepeatedCompositeFieldContainer, RepeatedScalarFieldContainer
 from google.protobuf.message import Message
 
 _ContainerMessageT = TypeVar("_ContainerMessageT", bound=Message)
-_ExtenderMessageT = TypeVar("_ExtenderMessageT", bound=Union[Message, bool, int, float, Text, bytes])
+_ExtenderMessageT = TypeVar(
+    "_ExtenderMessageT",
+    bound=Union[Message, RepeatedScalarFieldContainer[Any], RepeatedCompositeFieldContainer[Any], bool, int, float, Text, bytes],
```

Comment:
Between "Types Args can't be generic in bound" and "RepeatedScalarFieldContainer is missing type args" - I wasn't sure how to densely express what I'm trying to do here to pyright.

Syntactically I want something using _ScalarV (eg `RepeatedScalarFieldContainer[_ScalarV]` or even `RepeatedScalarFieldContainer[_]`)
https://github.com/python/typeshed/blob/master/stubs/protobuf/google/protobuf/internal/containers.pyi#L24

Practically, I either have to use the `Any` or write out all the possibilities as so
```
Union[RepeatedScalarFieldContainer[int], RepeatedScalarFieldContainer[bool], RepeatedScalarFieldContainer[Text], ...]
```

Any recommendation here?

mypy seems to be happy w/o the type arg to RepeatedScalarFieldContainer

Reply:
I think Any is appropriate here.